### ISSUE TITLE
AnimatedList for Enhancement.

### ISSUE THREAD

**Issue by icnahom** [Dimibe/grouped_list#35]
What are the possibilities of implementing `AnimatedList` to the package? If there is a possibly please consider it.

**Comment by Dimibe** [comment]
@icnahom Yeah would be nice to have this. Currently I'm very busy but great idea!

**Comment by cliftonlabrum** [comment]
When I found this package I was also hoping that it would support `AnimatedList`. 😄
